src/libOpenImageIO/imagebufalgo_opencv.cpp

```diff
@@ -136,7 +137,161 @@ IplImage *
 ImageBufAlgo::to_IplImage (const ImageBuf &src)
 {
 #ifdef USE_OPENCV
-    return NULL;
+    ImageSpec spec = src.spec();
+    ImageBuf tmp = src;
+
+    // If the image has 4+ channels, then reduce to 4 channels, and use BGRA
+    // order (OpenCV ordering).
+    if (spec.nchannels >=3) {
+        int channelorder[spec.nchannels];
+
+        // Set the channel order
+        for (int i = 0; i < spec.nchannels; ++i) {
+            channelorder[i] = i;
+        }
+
```

Comment:
Yeah, I do that, too, it's just a tradeoff about chunk size. My view is that the occasional blank line to separate quite different tasks within the same function, each of which may be a dozen or more lines, can be very helpful helpful, whereas if the whitespace is coming every 2-3 lines, it's too granular and the space isn't really adding to understanding, and in fact may hinder understanding by spreading out the code so that less is viewable in a single screenful, if you know what I mean.

This is purely a personal stylistic preference. I'm not claiming that your way is invalid or illogical. I don't even feel strongly enough about it to insist that you do it. Just pointing how to nudge it slightly toward the prevailing style of the codebase, if you are so inclined.